_data/3.11-manual.yml

```diff
@@ -207,119 +166,47 @@
-        - text: SmartGraph Management
-          href: graphs-smart-graphs-management.html
-    - text: EnterpriseGraphs
-      href: graphs-enterprise-graphs.html
-      children:
-        - text: Getting Started with EnterpriseGraphs
-          href: graphs-enterprise-graphs-getting-started.html
-        - text: EnterpriseGraph Management
-          href: graphs-enterprise-graphs-management.html
-    - text: Testing Graphs on Single Server
-      href: smart-and-satellite-graphs-single-server.html
-    - text: Traversals
-      href: graphs-traversals.html
-      children:
-        - text: Example Data
-          href: graphs-traversals-example-data.html
-    - text: Working with Edges
-      href: graphs-edges.html
-- text: Data Science
-  href: data-science.html
-  children:
-    - text: ArangoGraph Notebooks
-      href: arangograph-notebooks.html
-    - text: Pregel
-      href: graphs-pregel.html
-      children:
-        - text: Pregel Algorithms
-          href: graphs-pregel-algorithms.html
-    - text: ArangoDB-NetworkX Adapter
-      href: data-science-networkx-adapter.html
-    - text: ArangoDB-DGL Adapter
-      href: data-science-dgl-adapter.html
-    - text: ArangoDB-cuGraph Adapter
-      href: data-science-cugraph-adapter.html
-    - text: ArangoDB-PyG Adapter
-      href: data-science-pyg-adapter.html
-    - text: ArangoRDF Adapter
-      href: data-science-arango-rdf-adapter.html
-    - text: Mapping RDF to Graphs
-      href: data-modeling-graphs-from-rdf.html  
-- subtitle: ADVANCED TOPICS
+    - text: Example Datasets
+      href: arangosearch-example-datasets.html
+    - text: Exact value matching
+      href: arangosearch-exact-value-matching.html
+    - text: Range Queries
+      href: arangosearch-range-queries.html
+    - text: Prefix matching
+      href: arangosearch-prefix-matching.html
+    - text: Case-sensitivity and Diacritics
+      href: arangosearch-case-sensitivity-and-diacritics.html
+    - text: Wildcard search
+      href: arangosearch-wildcard-search.html
+    - text: Full-text token search
+      href: arangosearch-fulltext-token-search.html
+    - text: Phrase and proximity search
+      href: arangosearch-phrase-and-proximity-search.html
+    - text: Faceted search
+      href: arangosearch-faceted-search.html
+    - text: Fuzzy search
+      href: arangosearch-fuzzy-search.html
+    - text: Geospatial search
+      href: arangosearch-geospatial-search.html
+    - text: Search highlighting
+      href: arangosearch-search-highlighting.html
+    - text: Nested search
+      href: arangosearch-nested-search.html
+    - text: Ranking
+      href: arangosearch-ranking.html
+    - text: Performance
+      href: arangosearch-performance.html
+    - text: <code>search-alias</code> Views Reference
+      href: arangosearch-views-search-alias.html
+    - text: <code>arangosearch</code> Views Reference
+      href: arangosearch-views.html
+- subtitle: TRANSFORM DATA 
+- text: Analyzers
```

Comment:
Not a fan of having a separate section about transforming data but only one chapter

Reply:
Indeed, not desirable but we don't have other content. A possible solution is to move it under "Index, search, transform data". We need to check if the subtitle is not too lengthy though.

Reply:
While Analyzers are a standalone feature, they are most often used in combination with ArangoSearch. We shouldn't treat it like a component of ArangoSearch as you can use it independently, but placing it next to the indexing and search content would be okay in my opinion. Finding a succinct title is hard, though.